### PR TITLE
Replace travis with github actions

### PR DESCRIPTION
**Pull request by luca-rath**
| Q | A
| --- | ---
| Bug fix? | no
| New feature? | yes
| BC breaks? | no
| Deprecations? | no
| Fixed tickets | fixes #issuenum
| Related issues/PRs | https://github.com/sulu/sulu/issues/5693
| License | MIT

#### What's in this PR?

Replace travis with github actions

### CODE REVIEW

File: Tests/Functional/Controller/TaskControllerTest.php [hunk 1]
```diff
@@ -310,7 +310,8 @@ public function testPut(
             '/api/tasks/' . $postData['id'],
             [
                 'handlerClass' => $handlerClass,
-                'entityId' => $postData['id'],
+                'entityId' => $postData['entityId'],
+                'taskId' => $postData['taskId'],
```
Comment (luca-rath) [inline Tests/Functional/Controller/TaskControllerTest.php:314]: @alexander-schranz Can you take a look at these changes?

File: Resources/config/serializer/Task.xml [hunk 2]
```diff
@@ -8,6 +8,7 @@
         </property>
         <property name="entityClass" type="string" expose="true" groups="fullTask,partialTask,api"/>
         <property name="entityId" type="string" expose="true" groups="fullTask,partialTask,api"/>
+        <property name="taskId" type="string" expose="true" groups="fullTask,partialTask,api"/>
```
Comment (luca-rath) [inline Resources/config/serializer/Task.xml:11]: @alexander-schranz Can you take a look at this change?

File: Tests/Functional/Controller/TaskControllerTest.php (hunk 1 above)
Comment (luca-rath) [reply Tests/Functional/Controller/TaskControllerTest.php:314]: Without those changes, a test fails (See https://github.com/sulu/SuluAutomationBundle/runs/1707833641)

Comment (luca-rath) [reply Tests/Functional/Controller/TaskControllerTest.php:314]: That test did also fail on travis (See https://travis-ci.org/github/sulu/SuluAutomationBundle/jobs/730997908)

**Review by wachterjohannes — APPROVED** [review]

**Pull request merged**